src/Microsoft.DotNet.Build.Tasks.Feed/src/GenerateBuildManifest.cs

```
@@ -70,6 +72,13 @@ public override bool Execute()
         {
             try
             {
+                PublishingInfraVersion targetPublishingVersion = PublishingInfraVersion.Legacy;
+
+                if (!string.IsNullOrEmpty(PublishingVersion)) 
+                {
+                    Enum.TryParse(PublishingVersion, ignoreCase: true, out targetPublishingVersion);
```

If this fails, we should probably throw an error. This would mean that the user tried to set the publishing version to an incorrect value